rust_snuba/src/processors/mod.rs

```diff
@@ -48,6 +52,45 @@ define_processing_functions! {
     ("GenericGaugesMetricsProcessor", "snuba-generic-metrics", generic_metrics::process_gauge_message),
 }
 
+lazy_static! {
+    static ref COGS_MAP: HashMap<&'static str, COGSLabel> = {
+        let mut m = HashMap::new();
+        m.insert(
+            "GenericCountersMetricsProcessor",
+            COGSLabel {
+                resource_id: "generic_metrics_processor_counters".to_string(),
+                app_feature: "counter".to_string(),
```

Comment:
what's the `app_feature` for? And why is it `counter`? Isn't this supposed to be stuff like "custom" (for ddm), "release health", etc?

Reply:
yes. will fix that part

Reply:
with the right fix for `app_feature` it now even harder to get the right label in the message processor unless the interface of processing function changes. the use_case_id is only known within the processing function. 

i think the whole model of accounting has to be rethought